projects/plugins/crm/src/js/components/automations-admin/index.tsx

```diff
@@ -1,15 +1,49 @@
-import { AdminSection, Col, Container } from '@automattic/jetpack-components';
+import { Button, AdminSection, Col, Container } from '@automattic/jetpack-components';
+import { useQuery } from '@tanstack/react-query';
+import { dispatch, useSelect } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
-import React from 'react';
+import axios from 'axios';
+// import { Contact, Message } from 'crm/state/inbox/types';
+import { Workflow } from 'crm/state/automations-admin/types';
+import { store } from 'crm/state/store';
+import { useCallback, useState } from 'react';
 import { Link, Routes, Route } from 'react-router-dom';
 import AdminPage from '../admin-page';
 import { BulkWorkflowActions } from './components/bulk-workflow-actions';
+import { EditModal } from './components/edit-modal';
 import { RedirectHome } from './components/redirect-home';
 import { WorkflowRow } from './components/workflow-row';
 import { WorkflowTable } from './components/workflow-table';
 import { workflowOne, workflowTwo } from './test/util/data';
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+declare let jpcrmAutomationsInitialState: any;
+
 export const AutomationsAdmin = () => {
+	const [ isModalOpen, setIsModalOpen ] = useState( false );
+
+	const closeModal = useCallback( () => {
+		setIsModalOpen( false );
+	}, [ setIsModalOpen ] );
+
+	useQuery( {
```

Comment:
This is just a placeholder to load test data. Some thought will need to be given to put this in an area where all REST API calls can be implemented as query hooks.